schemas/blocks/calculated_summary.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "type": {
+        "type": "string",
+        "enum": [
+          "CalculatedSummary"
+        ]
+      },
+      "titles": {
+        "$ref": "../common_definitions.json#/titles"
+      },
+      "routing_rules": {
+        "$ref": "../common_definitions.json#/routing_rules"
+      },
+      "skip_conditions": {
+        "$ref": "../common_definitions.json#/skip_conditions"
+      },
+      "calculation": {
+        "type": "object",
+        "properties": {
+          "calculation_type": {
+            "type": "string",
+            "enum": [
+              "sum"
+            ]
+          },
+          "answers_to_calculate": {
+            "type": "array",
+            "minItems": 2,
```

Comment:
In the scenario that you're testing a survey part way through building it, is it conceivable that you'd run a survey where you've used a calculate block, but only have a single answer id in this block at the point you want to run a test of your other logic? My point is that it's probably a bit silly to only have one item in `answers_to_calculate`, but perhaps it shouldn't be invalid?

Reply:
I agree in that situation you may want just one answer but I believe schema validator is to protect against poorly designed schemas and to do that we have to treat each survey as complete at the point of validation. If you only need one answer then you should use the Confirmation block type and change it after.

Reply:
Okay @dwyeradam, I'll back off on that. I do wonder if having the ability to raise "Warnings/Recommendations" as well as hard errors may be useful in future?

Reply:
@benhowes Only back off if you're happy with my response. Otherwise we open it up to the group and hope for some consensus. Often with varying success. I won't take it personally.